src/server/blockchain/AdminWallet.js

```diff
@@ -78,91 +80,81 @@ export class Wallet {
     return web3Provider
   }
 
-  async init() {
+  async init(): Promise<any> {
     log.debug('Initializing wallet:', { conf: conf.ethereum })
-
-    this.web3 = new Web3(this.getWeb3TransportProvider(), null, {
-      defaultBlock: 'latest',
-      defaultGas: 200000,
-      defaultGasPrice: 1000000,
-      transactionBlockTimeout: 5,
-      transactionConfirmationBlocks: 1,
-      transactionPollingTimeout: 30
-    })
-    if (conf.privateKey) {
-      let account = this.web3.eth.accounts.privateKeyToAccount(conf.privateKey)
-      this.web3.eth.accounts.wallet.add(account)
-      this.web3.eth.defaultAccount = account.address
-      this.address = account.address
-      log.debug('Initialized by private key:', account.address)
-    } else if (conf.mnemonic) {
-      let root = HDKey.fromMasterSeed(bip39.mnemonicToSeed(this.mnemonic))
-      var path = "m/44'/60'/0'/0/0"
-      let addrNode = root.derive(path)
-      let account = this.web3.eth.accounts.privateKeyToAccount('0x' + addrNode._privateKey.toString('hex'))
-      this.web3.eth.accounts.wallet.add(account)
```

Comment:
here just loop over the paths and add the accounts
no need to copy the code from dapp